rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel="1.77.0"
```

Comment:
I feel that it best not to pin in one version?
Or using https://doc.rust-lang.org/cargo/reference/manifest.html?#the-rust-version-field

Reply:
It was locked to a version in a previous PR: https://github.com/containerd/ttrpc-rust/pull/179.  Whic is causing the failures in https://github.com/containerd/ttrpc-rust/issues/219

Using the toolchain file doesn't force that version, it just determines the toolchain to use if not already overridden.  It be overridden via other commands: https://rust-lang.github.io/rustup/overrides.html. 

I don't think the `rust-version` field helps with tool chain installation, but we can certainly set it as a minimal version.  Any ideas on the what version it should be set to?

Reply:
added the min `rust-version`

Reply:
Thanks, LGTM.
The minnest version is as the same `home v0.5.9`.
```
error: package `home v0.5.9` cannot be built because it requires rustc 1.70.0 or newer, while the currently active rustc version is 1.66.1
```